### PR TITLE
Refactor: Remove Dependencies from Migration v111

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

File: models/migrations/v111.go [hunk 1]
```diff
@@ -5,35 +5,373 @@
...
-	}
 
-	sess := x.NewSession()
-	defer sess.Close()
+		ReviewerID int64 `xorm:"index"`
+		IssueID    int64 `xorm:"index"`
+	}
 
-	if err := sess.Sync2(new(ProtectedBranch)); err != nil {
+	if err := x.Sync2(new(ProtectedBranch)); err != nil {
```
Comment (6543) [inline models/migrations/v111.go:40]: prevent sqlite deadlocks

from time to time if you Alter a table in sqlite you get locks when writting stuff to it

**Review by lafriks — APPROVED** [review]

**Comment by 6543** [comment]
blocking #12282

File: models/migrations/v111.go [hunk 2]
```diff
@@ -5,35 +5,361 @@
...
+	}
+
+	// Team represents a organization team.
+	type Team struct {
+		ID        int64 `xorm:"pk autoincr"`
+		OrgID     int64 `xorm:"INDEX"`
+		Authorize int
+	}
+
+	// getUserRepoPermission static function based on 5d78792385
```
Comment (6543) [inline models/migrations/v111.go]:
```suggestion
	// getUserRepoPermission static function based on models.IsOfficialReviewer at 5d78792385
```

**Review by mrsdizzie — APPROVED** [review]

**Comment by techknowlogick** [comment]
🚀

**Pull request merged**